### PR TITLE
Mquezada/m corr with ev

### PR DESCRIPTION
**Pull request by manuel-quezada**
Add few changes to MCorr model to accommodate the edge based (stabilization) methods, add a test to the full LS algorithm and other minor things.

### CODE REVIEW

**Comment by manuel-quezada** [comment]
@cekees can you take a look at this PR? It is basically some minor needed additions to MCorr to handle a projection from the conservative level set at quad points to the nodes.

File: proteus/mprans/VOF.py [hunk 1]
```diff
@@ -416,18 +418,18 @@ def attachModels(self,modelList):
             #
         #
     def initializeElementQuadrature(self,t,cq):
-        if self.flowModelIndex is None:
+        if self.flowModelIndex == None:
```
Comment (cekees) [inline proteus/mprans/VOF.py]: switch to "is None", see e.g. http://jaredgrubb.blogspot.com/2009/04/python-is-none-vs-none.html

Comment (zhang-alvin) [reply proteus/mprans/VOF.py]: I can take care of the file management/version control. The .h5 file will need to be purged from the git history to keep the repo size small.

**Comment by zhang-alvin** [comment]
I can take care of the file management/version control. The .h5 file will need to be purged from the git history to keep the repo size small.

**Comment by zhang-alvin** [comment]
Before this gets merged in, one thing that needs to be done is that the reference files for the new tests (i.e. levelset, griffiths,etc.) need to be added explicitly to `setup.py` and the testing directory needs to be added as a package. Otherwise, the tests will run locally but I don't think they'll run on Travis.

For example in setup.py:63: (package)
```
      packages = ['proteus',
                  'proteus.mprans',
                  'proteus.richards',
                  'proteus.elastoplastic',
                  'proteus.mbd',
                  'proteus.test_utils',
                  'proteus.config',
                  'proteus.tests',
                  'proteus.tests.ci',
                  'proteus.tests.matrix_constructor',
                  'proteus.tests.matrix_constructor.import_modules',
                  'proteus.tests.mesh_tests',
                  'proteus.tests.mesh_tests.import_modules',
                  'proteus.tests.linalgebra_tests',
                  'proteus.tests.post_processing',
                  'proteus.tests.post_processing.import_modules',
                  'proteus.tests.solver_tests_slow',
                  'proteus.tests.solver_tests_slow.import_modules',
                  'proteus.tests.matrix_constructor',
                  'proteus.tests.matrix_constructor.import_modules',
                  'proteus.tests.single_phase_gw',
                  'proteus.tests.poisson_2d',

```
and in setup.py:468: (test files)
```
      data_files=[(proteus_install_path,
                   ['proteus/proteus_blas.h',
                    'proteus/proteus_lapack.h',
                    'proteus/ModelFactory.h',
                    'proteus/CompKernel.h'
                   ]),
                  (os.path.join(proteus_install_path,'tests'),
                   ['proteus/tests/hex_cube_3x3.xmf',
                    'proteus/tests/hex_cube_3x3.h5']),
```

**Comment by zhang-alvin** [comment]
It looks like there are .h5 files not tracked by git-lfs even in the master branch. I tried to migrate all of the h5 files to git-lfs and purge them from the git history in this branch, but it created many conflicts that required manual merging. It would be more streamlined to merge this branch in and then address all of the .h5 files and their histories in the master branch.

**Comment by cekees** [comment]
Thanks @zhanga-alvin understood. I’ve done some of the test additions on the coverage branch that I merged. I’ll merge this branch and then we can fix master.

**Pull request merged**